superscore/tests/test_views.py

```diff
@@ -32,6 +37,42 @@ def pv_poll_model(
     qtbot.wait_until(lambda: not model._poll_thread.isRunning())
 
 
+@pytest.fixture(scope="function")
+def pv_table_view(
+    mock_client: Client,
+    simple_snapshot: Collection,
+    qtbot: QtBot,
+):
+    """
+    LivePVTableView, holds three PVs with different types.  Stored data allowed.
+    Mocks control layer to return realistic-ish EpicsData
+
+    TODO: add string field examples once we properly handle strings
+    """
+    # Build side effect function:
+    ret_vals = {
+        "MY:FLOAT": EpicsData(data=0.5, precision=3,
+                              upper_ctrl_limit=2, lower_ctrl_limit=-2),
+        "MY:INT": EpicsData(data=1, upper_ctrl_limit=10, lower_ctrl_limit=-10),
```

Comment:
The most nitpick I've ever nitpicked: high, low ranges break my brain and I'd prefer low, high
```suggestion
                              lower_ctrl_limit=-2, upper_ctrl_limit=2),
        "MY:INT": EpicsData(data=1, lower_ctrl_limit=-10, upper_ctrl_limit=10),
```